common/assets/scss/overrides/_govuk-frontend.scss

```diff
@@ -32,6 +32,27 @@
   padding: govuk-spacing(1) govuk-spacing(2);
 }
 
+// Add support to show when a field contains a message
+.govuk-form-group--message:not(.govuk-form-group--error) {
+  padding-left: govuk-spacing(3);
+  border-left: 5px solid govuk-colour("blue");
+
+}
+.govuk-prefill-message {
```

Comment:
Why not `.app-prefill-message`?

Reply:
I was actually a little torn about where to put this...let me have another stab at a name.